core/build.gradle

```diff
@@ -19,8 +19,10 @@ apply plugin: 'biz.aQute.bnd.builder'
 
 dependencies {
   api project(':api')
-  api "org.terracotta:statistics:$parent.statisticVersion"
+  implementation "org.terracotta:statistics:$parent.statisticVersion"
+  implementation "org.terracotta.management:management-model:$terracottaPlatformVersion"
   compileOnly 'org.osgi:osgi.core:6.0.0'
+  compileOnly 'org.osgi:org.osgi.service.component.annotations:1.3.0'
```

Comment:
This shouldn't be necessary.

Reply:
Its needed because `DefaultStatisticsServiceFactory` is a component